lib/routes/pollingRoutes.js

```diff
@@ -23,6 +24,26 @@ const createTerms = terms => (terms.active && terms.text) ? ({
   cancel: terms.cancelButtonText
 }) : null
 
+const buildTriggers = async (allTriggers) => {
+  const normalTriggers = []
+  const customTriggers = _.filter(o => {
+    if (o.customRequirementId === '') normalTriggers.push(o)
+    return o.customRequirementId !== ''
+  }, allTriggers)
+  const customTriggerDetails = await _.flow([_.map(_.get('customRequirementId')), customRequestQueries.batchGetCustomInfoRequest])(customTriggers)
+
+  customTriggerDetails.forEach((details, index) => {
+    customTriggers[index] = { ...customTriggers[index], customInfoRequest: details }
+  })
```

Comment:
```suggestion
  _.forEach((details, index) => {
    customTriggers[index] = { ...customTriggers[index], customInfoRequest: details }
  }, customTriggerDetails)
```

Reply:
Lodash doesn't support "index" on forEach

Reply:
You should have, according to the documentation

https://lodash.com/docs/4.17.15#forEach
`The iteratee is invoked with three arguments: (value, index|key, collection)`

Reply:
as we talked after this, lodash/fp does not support index after all